src/utils/convertCases/convertCases.ts

```diff
@@ -1,8 +1,13 @@
 export function convertCases(add: string): string {
-    return add
-        .replace('+2', '+Gen.')
-        .replace('+3', '+Dat.')
-        .replace('+4', '+Acc.')
-        .replace('+5', '+Ins.')
-        .replace('+6', '+Loc.');
+    const caseInfo = add
+        .replace('+1', '+Nom')
+        .replace('+2', '+Gen')
+        .replace('+3', '+Dat')
+        .replace('+4', '+Acc')
+        .replace('+5', '+Ins')
+        .replace('+6', '+Loc')
+        .replace('+7', '+Voc')
+        .replace('.', ''); 
+    
+    return ['+Nom','+Gen','+Dat','+Acc','+Ins','+Loc','+Voc'].find((el) => caseInfo.indexOf(el) !== -1) ? caseInfo : '';
```

Comment:
Doesn't look good enough.

Better create a set outside, e.g. `const VALID_VALUES = new Set(['+Nom', '+Gen', ...])` and check `VALID_VALUES.has(caseInfo) ? caseInfo : ''`

Reply:
Done in a similar way.